content/articles/use-geopy-to-calculate-the-distance-between-two-points./index.md

```diff
@@ -0,0 +1,188 @@
+### The Geopy method for distance calculation between any two points on the globe.
+### Introduction.
+Geopy is essentially a Python library that makes the calculation of geographic distances easier. It becomes considerably easier for developers using Geopy to retrieve the coordinates of various sites by leveraging third-party geocoders and other data sources.
+
+In this article, we'll look at a few alternative ways for calculating the distance between any two points on the globe. We shall dig deep to explain how this is possible using python package geopy which is the primary python package for distance calculation.
+
+The user needs first perform the following command to install geopy.
+
+Before executing the command below, there is a series of steps that are followed so that the command is finally arrived at.
+1. Open your cmd and make sure it is running as an administrator.
+2. Make sure that you are operating from the root by performing the following. cd..
+3. cd.. This shows that you are already at the root.
+4. After that now you can perform the command below and geopy will be installed in your machine.
+```python
+pip install geopy
+```
+We are therefore set to calculate the distances of any two points following a successiful installation of the geopy module.
+
+Using the Geopy, we can calculate or find the distance between two strategic positions using the various methods that are listed below.
+1. Distance Measure using the Geodesic Measure.
+2. Distance Measure using The greate circle distance.
+3. Distance Measure using The Herversine formula.
+### The Geodesic Measure for distance calculation.
+The geodesic measure or distance is used to define the shortest path between any two given point on the globe.
+
+As in the situation of two diametrically opposing points on a sphere, it is feasible that numerous alternative arcs between two points reduce the distance. In this case, any of these curves is a geodesic. Geodesics are not exactly the same as " the shortest curves" between any two given locations, despite their likeness.
+
+Geodesics, on the other hand, are just the shortest distance between two points at a given position that is parameterized with "constant speed." A geodesic is the "long way around" on a great circle between two points on an earth's surface, even if it is not the shortest path between the two points.
+
+An important point to note when working with geodesic or when calculating any distance between any two points on the sphere is that it does not ignore any curved part. Geodesic distance measures distance in such a way that it includes even the curved part. In contrast to this, a distance calculation known as Euclidean distance completely ignores the shape of the path that is involved such that it assumes a straight path from the start of a point to its end.
```

Comment:
This sentence `Geodesic distance measures distance in such a way that it includes even the curved part` appears to be a repetition.